akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -272,11 +272,11 @@ final case class RootActorPath(address: Address, name: String = "/") extends Act
 
   override val toSerializationFormat: String = toString
 
-  override def toStringWithAddress(addr: Address): String =
+  override def toStringWithAddress(newAddress: Address): String =
```

Comment:
These are actually not source compatible, since the parameter names in Scala are a part of the API (because you could use named params, not likely that anyone does for this but we cannot know nobody did a `toStringWithAddress(addr = something)` which will not work anymore with this change.

Strange that MiMa didn't complain about that.